content/tutorials/webrtc/infrastructure/en/index.html

```diff
@@ -38,51 +38,42 @@
 
-  <p>BUT...</p>
+<p>This article shows you how to build a signaling service, and how to deal with the quirks of real-world connectivity with STUN and TURN servers. It also explains how WebRTC apps can handle multiparty calls and interact with services, such as VoIP and PSTN (also known as telephones).</p>
 
-  <p>WebRTC still needs servers:</p>
-
-  <ul>
-    <li>For clients to exchange metadata to coordinate communication: this is called signaling.</li>
-    <li>To cope with network address translators (NATs) and firewalls.</li>
-  </ul>
-
-<p>In this article we show you how to build a signaling service, and how to deal with the quirks of real-world connectivity by using STUN and TURN servers. We also explain how WebRTC apps can handle multi-party calls and interact with services such as VoIP and PSTN (aka telephones).</p>
-
-<p>If you're not familiar with the basics of WebRTC, we strongly recommend you take a look at <a href="http://www.html5rocks.com/en/tutorials/webrtc/basics/">Getting Started With WebRTC</a> before reading this article.</p>
+<p>If you're not familiar with the basics of WebRTC, see <a href="http://www.html5rocks.com/en/tutorials/webrtc/basics/">Getting Started With WebRTC</a> before you read this article.</p>
 
 </blockquote>
 
 <h2 id="what-is-signaling">What is signaling?</h2>
 
-<p>Signaling is the process of coordinating communication. In order for a WebRTC application to set up a 'call', its clients need to exchange information:</p>
+<p>Signaling is the process of coordinating communication. In order for a WebRTC app to set up a call, its clients need to exchange information, such as the following:</p>
 
 <ul>
-  <li>Session control messages used to open or close communication.</li>
-  <li>Error messages.</li>
-  <li>Media metadata such as codecs and codec settings, bandwidth and media types.</li>
-  <li>Key data, used to establish secure connections.</li>
-  <li>Network data, such as a host's IP address and port as seen by the outside world.</li>
+  <li>Session-control messages used to open or close communication</li>
+  <li>Error messages</li>
+  <li>Media metadata, such as codecs, codec settings, bandwidth, and media types</li>
+  <li>Key data used to establish secure connections</li>
+  <li>Network data, such as a host's IP address and port as seen by the outside world</li>
 </ul>
 
-<p>This signaling process needs a way for clients to pass messages back and forth. That mechanism is not implemented by the WebRTC APIs: you need to build it yourself. We describe below some ways to build a signaling service. First, however, a little context...</p>
+<p>This signaling process needs a way for clients to pass messages back and forth. That mechanism is not implemented by the WebRTC APIs. You need to build it yourself. You read about some ways to build a signaling service later in this article. First, however, you need a little context.</p>
```

Comment:
Maybe 'Later in this article you will will learn ways to build a signaling service. Fist, however, you need a little context.'

Reply:
Will fix

Reply:
Will fix